infer.py

```diff
@@ -27,16 +28,18 @@ def main():
         transforms.ToTensor(),
         transforms.Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5))
     ])
-    fig, ax = plt.subplots(2, 6, figsize=(20, 10))
+    nr_images = len(image_paths)
+    fig, ax = plt.subplots(2, nr_images, figsize=(20, 10))
     random.shuffle(image_paths)
-    for i in range(6):
+    for i in range(nr_images):
```

Comment:
This is not ideal if the number of images is 1000....on one plot, it will never be viewable or fit.

Reply:
Just thought it made more sense than forcing the folder to have exactly 6 images. changed it to:
```nr_images = len(image_paths) if len(image_paths) < 6 else 6```
so that it works for <=6 and if there are more than 6 then it caps it off.